### PR TITLE
FIRECERT-1715: FCA is unable to unregister from Listeners for rpcOnly…

### PR DESCRIPTION
**Pull request by anjalimukundan**
…/Transport

# Description
Enter a detailed description of the PR here, including the nature and purpose of the changes.

## JIRA Ticket
Include a link to the JIRA Ticket linked to this PR.
[Issue-0000](https://your-jira-domain.com/browse/Issue-0000)

## Additional Info (Test steps / Steps to reproduce etc.)
Details on how to test the feature

### CODE REVIEW

**Review by neeradanelxsi — APPROVED** [review]

File: src/EventInvocation.js [hunk 1]
```diff
@@ -239,7 +239,7 @@ export class EventInvocation {
 
   // This method will clear the eventListeners and the event hsitory for the listener as a part of FCA
   clearAllListeners() {
-    logger.info('Going to clear  registering listeners' + eventHandlerMap, 'clearAllListeners');
+    logger.info('Going to clear registering listeners' + JSON.stringify(eventHandlerMap), 'clearAllListeners');
```
Comment (parag-pv) [inline src/EventInvocation.js:242]: Rephrase to `Clearing registered listeners`

File: src/EventInvocation.js [hunk 2]
```diff
@@ -248,9 +248,15 @@ export class EventInvocation {
           const eventName = EventHandlerObject.eventName;
           const eventRegistrationID = uniqueListenerKey.split('-')[1];
           const [sdkType, module] = this.getSdkTypeAndModule(eventNameWithModuleName);
-          // going to unregister the event listener by registrationID
+          // going to unregister the event listener by eventName
```
Comment (parag-pv) [inline src/EventInvocation.js:251]: Add additional documentation for how listeners are cleared between SDK and Transport modes

File: src/EventInvocation.js (hunk 1 above)
Comment (anjalimukundan) [reply src/EventInvocation.js]: Addressed the comment

File: src/EventInvocation.js (hunk 2 above)
Comment (anjalimukundan) [reply src/EventInvocation.js]: Addressed the comment

**Review by parag-pv — APPROVED** [review]

**Pull request merged**